scripts/ci/run-scale.sh

```diff
@@ -12,15 +12,20 @@ cd ${GOPATH}/src/github.com/skydive-project/skydive
 make install
 
 export SKYDIVE_ANALYZERS=localhost:8082
-export ELASTICSEARCH=localhost:9200
+export ELASTICSEARCH=localhost:9201
 export TLS=true
 export SKYDIVE=${GOPATH}/bin/skydive
 export FLOW_PROTOCOL=${FLOW_PROTOCOL:-websocket}
 export SKYDIVE_LOGGING_LEVEL=DEBUG
 
+docker run -d --name elasticsearch  -p 9201:9200 -p 9301:9300 -e "discovery.type=single-node" elasticsearch:7.7.1
+
 make test.functionals WITH_SCALE=true TAGS=${TAGS} VERBOSE=true TIMEOUT=10m TEST_PATTERN=Scale EXTRA_ARGS="-logs=/tmp/skydive-scale/scale.log"
 status=$?
 
+docker stop elasticsearch
```

Comment:
function es_cleanup()
{
    docker stop elasticsearch
    docker rm elasticsearch
}

trap es_cleanup EXIT

Reply:
This could be put in `scripts/ci/run-tests-utils.sh` like :
```
function es_setup()
{
    docker run -d --name elasticsearch  -p 9201:9200 -p 9301:9300 -e "discovery.type=single-node" elasticsearch:7.7.1 
}

function es_cleanup()
{
    docker stop elasticsearch
    docker rm elasticsearch
}

if [ "$BACKEND" = "elasticsearch" ]; then
    es_setup
    trap es_cleanup EXIT
fi
```